media/js/app/projects/sequenceAssignmentView.js

```diff
@@ -61,6 +62,19 @@
             if (options.isFaculty) {
                 jQuery(window).bind('beforeunload', this.beforeUnload);
             }
+
+            this.mapSignals();
+        },
+        mapSignals: function() {
+            var self = this;
+
+            jQuery(window).on('sequenceassignment.save', function(e) {
+                self.onSaveProject();
```

Comment:
I was envisioning it the other way around, but the way you've described makes sense. I will change this from a listener to a trigger and call it in `onSaveProject.success`.